app/code/Magento/ProductAlert/Block/Adminhtml/Catalog/Product/Edit/Tab/Alerts/Renderer/Stock.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Magento\ProductAlert\Block\Adminhtml\Catalog\Product\Edit\Tab\Alerts\Renderer;
+
+use Magento\Backend\Block\Widget\Grid\Column\Renderer\AbstractRenderer;
+use Magento\Framework\DataObject;
+use Magento\Framework\Message\ManagerInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Backend\Block\Context;
+use Magento\InventorySales\Model\StockResolver;
+use Magento\InventorySalesApi\Api\Data\SalesChannelInterface;
+
+class Stock extends AbstractRenderer
+{
+    /**
+     * @var StoreManagerInterface
+     */
+    protected $_storeManager;
+
+    /**
+     * @var StockResolver
+     */
+    protected $_stockResolver;
+
+    /**
+     * @var ManagerInterface
+     */
+    protected $_messageManager;
+
+    /**
+     * Stock constructor.
+     * @param Context $context
+     * @param StoreManagerInterface $storeManager
+     * @param StockResolver $stockResolver
+     * @param ManagerInterface $messageManager
+     * @param array $data
+     */
+    public function __construct(
+        Context $context,
+        StoreManagerInterface $storeManager,
+        StockResolver $stockResolver,
+        ManagerInterface $messageManager,
+        array $data = []
+    )
+    {
+        parent::__construct($context, $data);
+        $this->_storeManager = $storeManager;
+        $this->_stockResolver = $stockResolver;
+        $this->_messageManager = $messageManager;
+    }
+
+    /**
+     * Handles Stock column
+     *
+     * @param DataObject $row
+     * @return string
+     */
+    public function render(DataObject $row)
+    {
+        try {
+            $value = $this->getColumnData($row);
+            return $value;
+        } catch (\Exception $exception) {
```

Comment:
In which cases can be an exception? Why do we ignore it?

Reply:
I'll remove that try catch construction. It doesn't have to be here